tests/fixtures/uaa_fixture.py

```diff
@@ -0,0 +1,217 @@
+import pytest
+from math import ceil
+import time
+from random import randint
+
+from fixtures.config import authenticated_response, base_url
+
+
+def get_epoch_days_ago(days_ago: int, current_time: float) -> int:
+    return int(current_time) - (days_ago * 60 * 60 * 24 * 1000)
+
+
+def build_url(path, start_of_day: int = None, end_of_day: int = None, **params):
+    params_list = []
+
+    if params:
+        for k, v in params.items():
+            params_list.append(f"{k}={v}")
+
+    params_string = "&".join(params_list)
+
+    if not start_of_day and not end_of_day:
+        if params_string:
+            return f"{base_url}{path}?{params_string}"
+        else:
+            return f"{base_url}{path}"
+
+    filter_origin = 'origin eq "cloud.gov"'
+    filter_active = "active eq true"
+    filter_last_logon = (
+        f"last_logon_success_time ge {start_of_day}"
+        f" and "
+        f"last_logon_success_time le {end_of_day}"
+    )
+
+    filter_params = f"{filter_origin} and {filter_active} and {filter_last_logon}"
```

Comment:
How do these `filter_params` work when being included in the URL?  They don't appear like regular `key=value` pairs to me, but I am likely missing something. :-)

Reply:
It's the [SCIM syntax](https://ldapwiki.com/wiki/SCIM%20Filtering) which is 🤷 , I should probably up the variable name to `scim_filter` to be more semantic 👍 .